src/assets-loader/assets-loader.spec.ts

```diff
@@ -90,55 +74,60 @@ describe('AssetsLoader', () => {
           );
           const assetsUpdateSupply = jest.spyOn(assetsService, 'updateSupply');
 
-          await assetsLoader.loadDescriptions(blockDto, prisma);
+          await assetsLoader.loadDescriptions(true, transactionDto);
           expect(assetsUpsert).toHaveBeenCalledTimes(1);
-          expect(assetsUpsert).toHaveBeenCalledWith(
-            {
-              identifier: mint.id,
-              metadata: mint.metadata,
-              name: mint.name,
-              owner: mint.owner,
-            },
-            transaction,
-            prisma,
-          );
+          expect(assetsUpsert.mock.calls[0][0]).toEqual({
```

Comment:
These mock calls look crazy, is it because of the graphile worker? Not a blocker for shipping if the tests pass but we might want to relabel afterwords

Reply:
It's due to Jest OOM object comparison. Going to leave as is for now